daemon/k8s_watcher.go

```diff
@@ -78,6 +78,18 @@ const (
 	k8sAPIGroupIngressV1Beta1   = "extensions/v1beta1::Ingress"
 	k8sAPIGroupCiliumV2         = "cilium/v2::CiliumNetworkPolicy"
 	cacheSyncTimeout            = time.Duration(3 * time.Minute)
+
+	metricCNP      = "CiliumNetworkPolicy"
+	metricEndpoint = "Endpoint"
+	metricIngress  = "Ingress"
+	metricKNP      = "KubernetesNetworkPolicy"
```

Comment:
This is still wrong. We should use the real name which is `NetworkPolicy` and not `KubernetesNetworkPolicy`.